onnxruntime/python/tools/transformers/models/whisper/whisper_helper.py

```diff
@@ -270,6 +269,18 @@ def verify_onnx(
         pt_model = WhisperForConditionalGeneration.from_pretrained(model_name_or_path).to(device)
         processor = WhisperProcessor.from_pretrained(model_name_or_path)
         config = WhisperConfig.from_pretrained(model_name_or_path)
+
+        # Try to import `datasets` pip package
+        try:
+            from datasets import load_dataset
+        except Exception as e:
```

Comment:
Need not handle importError here. requirements.txt shall add datasets?

Reply:
The `datasets` package is only used during parity verification. Since it is otherwise not required for running Whisper, the import error is handled here instead of creating `requirements.txt`.

Reply:
The `datasets` package is only used during parity verification. Since it is otherwise not used when running Whisper, the import error is handled here instead of creating `requirements.txt`.

Reply:
The `datasets` package is only used during parity verification. Since it is otherwise not used when running Whisper, the import error is handled here instead of creating `requirements.txt`. We can change this and add it to `requirements.txt` in the other PR with the Whisper scripts if needed.